07/hangman/.gitignore

```diff
@@ -0,0 +1,8 @@
+ai.cpython-310.pyc
+hangman.cpython-310.pyc
+picture.cpython-310.pyc
+pictures.cpython-310.pyc
+process.cpython-310.pyc
+test_hangman.cpython-310-pytest-7.2.0.pyc
```

Comment:
Uf, no místo toho, abys v každém adresáři každé lekce vypisovala každý `*.pyc` soubor, který ti Python může vygenerovat, bys mohla ***do rootu celého repozitáře*** do jednoho nejobecnějšího `.gitignore` vložit prostě tohle:


```gitignore
*.pyc 
```

A měla bys to obecně. 😛